content/3_Silver/Intro_Sorted_Sets.mdx

```diff
@@ -125,13 +123,53 @@ which can only be used once before calling the `next()` method.
 </Warning>
 
 </JavaSection>
+<PySection>
+
+In Python, we can use `iter()` to obtain the iterator object of any iterable.
+Using `next()` on the iterator lets you iterate through the iterable.
+
+Iterating through a `set`:
+
+```py
+nums = {1, "three", 0, (1, 3, 5), 4}
+iterator = iter(nums)
+
+print(next(iterator))  # 0
+print(next(iterator))  # 1
+print(next(iterator))  # 4
+print(next(iterator))  # three
+print(next(iterator))  # (1, 3, 5)
+
+# Note:
+# The numbers appear sorted when iterating due to the hashing implementation
+# Sets are not ordered, so do not expect ordered/sorted behavior
+```
+
+Iterating through a `SortedSet`:
```

Comment:
i don't think you need this because you go through it later--it's not like they can use this on usaco anyways 👀

Reply:
Oops, I think Kevin already mentioned this.

Reply:
Got it!